sdk/go/common/tokens/names.go

```diff
@@ -36,7 +36,7 @@ var nameRestCharRegexp = regexp.MustCompile("^" + nameRestCharRegexpPattern + "$
 var NameRegexpPattern = nameFirstCharRegexpPattern + nameRestCharRegexpPattern
 
 const nameFirstCharRegexpPattern = "[A-Za-z_.]"
-const nameRestCharRegexpPattern = `[A-Za-z0-9_.]*`
+const nameRestCharRegexpPattern = `[A-Za-z0-9_.-]*`
```

Comment:
Comment on line 24 and 53 needs updating to match this

Reply:
Comments updated to reference `NameRegexpPattern`